### DISCUSSION TITLE
Break apart alerts with conditions

### DISCUSSION THREAD

**Discussion by bck01215** [grafana/grafana#57859] [Alerting]
### When

When creating alerts in Grafana and using a condition, alerts get grouped into one. 
![image](https://user-images.githubusercontent.com/77836661/198853245-71788fdd-2bcc-4f9f-af8a-7511e600aaba.png)

When using the query to generate the alerts, I see the expected behavior of multiple alerts with unique labels

![image](https://user-images.githubusercontent.com/77836661/198853321-ae0981e8-c7d7-4ed3-be13-8f219bd81809.png)

On datasources like prometheus, this isn't problematic as it's simple to add an expression on the end. For datasources without this ability, it's impossible to create dynamic alerts for each unique label. Maybe adding an aggregator to the condition would help.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
